react-form-controls/my-app/src/App.js

```diff
@@ -0,0 +1,19 @@
+import './App.css';
+import RegistrationFormUncontrolled from './RegistrationFormUncontrolled';
+import RegistrationFormControlled from './RegistrationFormControlled';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <h1>Registration Form (Uncontrolled)</h1>
+        <RegistrationFormUncontrolled />
+
+        <h1>Registration Form (Controlled)</h1>
```

Comment:
Only 1 `h1` element should be used per page. I would probably turn these into `h2` elements.